tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/PortalRuntime.java

```diff
@@ -50,6 +67,85 @@ public void dispose()
         }
     }
 
+    private IPath findBundledJREPath( IPath location )
```

Comment:
I don't want to support this feature anymore of autowiring the bundle jre, it causes too many bugs and i don't think it is useful going forward.  thanks.